### PR TITLE
fix/#285 - Lint errors and name fixes, also add owners for home-service

### LINKED ISSUE #285: Lint errors and name fixes, also add owners for home-service
## Is your chore related to a problem? Please describe

Variable name change and lint fixes for home-service

## Describe the solution you'd like

The home-service shouldn't have lint errors and better response names
 
## Describe alternatives you've considered

None

### PR DESCRIPTION
**Pull request by hybridx**
fix/#285 - Added owners array to types and also added data to update home type

## Explain the feature/fix

Added owners to types and fixed naming errors
```
Lint errors are possibly due to not installing node_modules
```

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

File: packages/home-service/src/resolver.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const HomeResolver = {
       .then(response => {
         return Object.assign(response, args.input)
           .save()
-          .then((user: any) => user);
+          .then((data: any) => data);
```
Comment (deshmukhmayur) [inline packages/home-service/src/resolver.ts:33]: The query resolvers should fetch the user details from the User/Group Microservice and stitch them back in the response.

Also, try to use a single query to fetch all the users required for the response.

Comment (hybridx) [reply packages/home-service/src/resolver.ts:33]: @deshmukhmayur that would be done in a different PR https://github.com/1-Platform/one-platform/pull/284

Comment (hybridx) [reply packages/home-service/src/resolver.ts:33]: This is to add the owners array

Comment (deshmukhmayur) [reply packages/home-service/src/resolver.ts:33]: That PR is mainly for the timestamps. But I guess the code change is similar. Ok.

**Pull request merged**